pkg/controller/operators/suite_test.go

```diff
@@ -47,15 +53,30 @@ var (
 		GracePeriodSeconds: &gracePeriod,
 		PropagationPolicy:  &propagation,
 	}
-	genName = names.SimpleNameGenerator.GenerateName
+	genName  = names.SimpleNameGenerator.GenerateName
+	fixtures = testobj.NewFixtureFiller(
```

Comment:
👍 

Aside - I've been meaning to write a library (or generator) that would let you perturb a fixture for certain tests.

```go
inputRoleBinding := fixtureRB.Set("metadata.name", "newName").Set("other.field", "value")
```